cvat-ui/src/components/consensus-management-page/task-consensus/consensus-settings-form.tsx

```diff
@@ -75,7 +75,7 @@ export default function ConsensusSettingsForm(props: Readonly<Props>): JSX.Eleme
                 <Text strong>General</Text>
                 <CVATTooltip
                     title={generalTooltip}
-                    className='cvat-settings-tooltip'
+                    className='cvat-management-page-tooltip'
```

Comment:
We still have many `cvat-settings-tooltip` in the code in this patch.

But, I think it should actually stay as is.
As not all tooltips on different management pages should follow such styles.
It is only about settings.

Reply:
And for some cards.. 
But for color maybe just define a common class e.g. `.cvat-white-color-tooltip`?

Reply:
So, as some cards are using this too, why we need 2 different places to specify white text?
Its easier to have one class

Reply:
After adding `cvat-management-page-tooltip` we also have 2 different places where white text specified. 
Not any tooltip from any management page should have these styles, so, it is more about the name. The previous was good enough

Reply:
Ok. Switched back